tests/benchdnn/graph/flex_rewrite.cpp

```diff
@@ -1329,6 +1330,57 @@ void flex_rewrite_t::dt_rewrite(deserialized_graph_t &dgraph) {
     }
 }
 
+void flex_rewrite_t::op_kind_rewrite(deserialized_graph_t &dgraph) {
+    // Step 1: check the op kind in the map and whether the given ids are in
+    // the graph.
+    for (const auto &v : op_kind_map_) {
+        if (v.second == "default") return;
+
+        auto target_kind = opstr2kind(v.second);
+        if (target_kind == dnnl::graph::op::kind::LastSymbol) return;
+
+        // Only support op kind rewrite for binary and eltwise ops.
+        auto target_driver = opkind2driver(target_kind);
+        if (target_driver != dnnl_driver_t::binary
+                && target_driver != dnnl_driver_t::eltwise) {
+            BENCHDNN_PRINT(0,
+                    "graph: rewrite: target op kind %s for id `%zd` is not "
+                    "supported\n",
+                    v.second.c_str(), v.first);
+            SAFE_V(FAIL);
+        }
+
+        bool found_id = false;
+        for (auto &aop : dgraph.ops_) {
+            if (aop.id_ == v.first) {
```

Comment:
```suggestion
        const auto &aop = dgraph.get_op(v.first);
```
Then check for it being `empty`. This way `found_id` is not needed.
And don't need to traverse in "step 2" as well, can replace right away.

Reply:
I removed `found_id` with the `get_op()` API of the deserialized graph, but as the returned aop is `const`, we still need step 2 for another traverse.